src/test/fuzz/ancestorpackage.cpp

```diff
@@ -0,0 +1,52 @@
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/script.h>
+#include <test/util/setup_common.h>
+
+#include <policy/packages.h>
+
+#include <set>
+#include <vector>
+
+namespace {
+FUZZ_TARGET(ancestorpackage)
+{
+    FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
+    std::vector<CTransactionRef> txns_in;
+    // Avoid repeat coins, as they may cause transactions to conflict
+    std::set<COutPoint> available_coins;
+    for (auto i{0}; i < 100; ++i) {
+        if (auto outpoint{ConsumeDeserializable<COutPoint>(fuzzed_data_provider)}) available_coins.insert(*outpoint);
```

Comment:
hmm I think the fuzzer was smart enough to give a coin with the same hash as the txid of a constructed transaction below, built on a different set of ancestors...

I think hashing these bytes would defeat this since it can't generate a valid tx with that txid?

Reply:
or to be lazier, just set the initial utxos' prevout.n index to some impossibly high numbers to ever be generated

Reply:
> I think hashing these bytes would defeat this since it can't generate a valid tx with that txid?

Did this one, thanks 👍